watchman/views.py

```diff
@@ -32,17 +32,31 @@ def _get_check_params(request):
 @json_view
 def status(request):
     response = {}
+    http_code = 200
 
     check_list, skip_list = _get_check_params(request)
 
     for check in get_checks(check_list=check_list, skip_list=skip_list):
         if callable(check):
-            response.update(check())
+            _check = check()
+            # Set our HTTP status code if there were any errors
+            if settings.WATCHMAN_ERROR_CODE:
```

Comment:
I think this needs to be updated since `settings.WATCHMAN_ERROR_CODE` is always truthy (an int) now.

`if settings.WATCHMAN_ERROR_CODE != 200:`